src/main/resources/i18n/messages_de.properties

```diff
@@ -98,5 +98,16 @@ email.notification.title.post.plagiarismVerdict=Entscheidung zum Plagiatsfall in
 email.notification.title.post.plagiarismCase=Neuer Plagiatsfall in Aufgabe {0}
 email.notification.aux.plagiarismVerdict.plagiarism=Der Fall wird als Plagiat angesehen!
 email.notification.aux.plagiarismVerdict.point.deduction=Wegen des Plagiatsfalls ziehen wir dir Punkte in der Aufgabe ab!
-email.notification.aux.plagiarismVerdict.warning=Du wirst wegen des Plagiatsversuchs verwarnt. Weitere Plagiate k�nnen schwere Folgen haben!
+email.notification.aux.plagiarismVerdict.warning=Du wirst wegen des Plagiatsversuchs verwarnt. Weitere Plagiate können schwere Folgen haben!
 email.notification.aux.plagiarismVerdict.no.plagiarism=Der Fall wird nicht mehr als Plagiat angesehen.
+
+# Tutorial Groups
+email.notification.title.tutorialGroup.registrationStudent=Du wurdest für die Übungsgruppe {0} im Kurs {1} registriert von {2}.
```

Comment:
See English messages file.

Reply:
Changed to active form in a2d98006db19207c790b367b9e6479bd55ba6c9d